Cargo.toml

```diff
@@ -55,31 +55,31 @@ version = "0.36.0"
-fuel-core-importer = { version = "0.35.0", path = "./crates/services/importer" }
-fuel-core-gas-price-service = { version = "0.35.0", path = "crates/services/gas_price_service" }
-fuel-core-p2p = { version = "0.35.0", path = "./crates/services/p2p" }
-fuel-core-producer = { version = "0.35.0", path = "./crates/services/producer" }
-fuel-core-relayer = { version = "0.35.0", path = "./crates/services/relayer" }
-fuel-core-sync = { version = "0.35.0", path = "./crates/services/sync" }
-fuel-core-txpool = { version = "0.35.0", path = "./crates/services/txpool" }
-fuel-core-txpool-v2 = { version = "0.35.0", path = "./crates/services/txpool_v2" }
-fuel-core-storage = { version = "0.35.0", path = "./crates/storage", default-features = false }
-fuel-core-trace = { version = "0.35.0", path = "./crates/trace" }
-fuel-core-types = { version = "0.35.0", path = "./crates/types", default-features = false }
+fuel-core = { version = "0.36.0", path = "./crates/fuel-core", default-features = false }
+fuel-core-client-bin = { version = "0.36.0", path = "./bin/fuel-core-client" }
+fuel-core-bin = { version = "0.36.0", path = "./bin/fuel-core" }
+fuel-core-keygen = { version = "0.36.0", path = "./crates/keygen" }
+fuel-core-keygen-bin = { version = "0.36.0", path = "./bin/keygen" }
+fuel-core-chain-config = { version = "0.36.0", path = "./crates/chain-config", default-features = false }
+fuel-core-client = { version = "0.36.0", path = "./crates/client" }
+fuel-core-database = { version = "0.36.0", path = "./crates/database" }
+fuel-core-metrics = { version = "0.36.0", path = "./crates/metrics" }
+fuel-core-services = { version = "0.36.0", path = "./crates/services" }
+fuel-core-consensus-module = { version = "0.36.0", path = "./crates/services/consensus_module" }
+fuel-core-bft = { version = "0.36.0", path = "./crates/services/consensus_module/bft" }
+fuel-core-poa = { version = "0.36.0", path = "./crates/services/consensus_module/poa" }
+fuel-core-executor = { version = "0.36.0", path = "./crates/services/executor", default-features = false }
+fuel-core-importer = { version = "0.36.0", path = "./crates/services/importer" }
+fuel-core-gas-price-service = { version = "0.36.0", path = "crates/services/gas_price_service" }
+fuel-core-p2p = { version = "0.36.0", path = "./crates/services/p2p" }
+fuel-core-producer = { version = "0.36.0", path = "./crates/services/producer" }
+fuel-core-relayer = { version = "0.36.0", path = "./crates/services/relayer" }
+fuel-core-sync = { version = "0.36.0", path = "./crates/services/sync" }
+fuel-core-txpool = { version = "0.36.0", path = "./crates/services/txpool" }
+fuel-core-txpool-v2 = { version = "0.36.0", path = "./crates/services/txpool_v2" }
+fuel-core-storage = { version = "0.36.0", path = "./crates/storage", default-features = false }
+fuel-core-trace = { version = "0.36.0", path = "./crates/trace" }
+fuel-core-types = { version = "0.36.0", path = "./crates/types", default-features = false }
```

Comment:
Unrelated to this PR, but why do we even specify versions here since these are local path dependencies? I'd suggest just removing the version argument completely.